requirements.txt

```diff
@@ -0,0 +1,5 @@
+Python libraries that need to be installed:
```

Comment:
This isn't a traditional `requirements.txt` file - it'll choke if someone actually tries `pip install -r requirements.txt`. Better to put it in the README or something if the extra pip parameters really are required.